doc/source/file_formats.rst

```diff
@@ -1,10 +1,23 @@
 File formats
 ============
-Here we will, at a later date, explain the exact syntax and semantics of the
-file formats.
+VerMoUTH introduces two new file formats. The ``.ff`` format for defining
```

Comment:
I would name that one is for the description of the force field (force field specific) and one is for conversion between force fields (mapping)

Reply:
That follows in that same paragraph though. Adding it here /again/ results in a lot of duplication